dpc-sdp/ripple-framework
# [R20-1666] add non-pii form data to submission events

**Issue**: https://digital-vic.atlassian.net/browse/R20-1666

### What I did

- Adds non-pii form data to submission events/update events

_Right now all editable fields are treated as potentially containing pii. So things like radio buttons, checkboxes and selects are marked as non-pii in the webform mapping. If the backend adds support for pii we can set the pii state on a field by field basis._

<img width="542" alt="Screenshot 2023-12-27 at 10 53 52 am" src="https://github.com/dpc-sdp/ripple-framework/assets/287178/f811b847-972f-4121-8245-e117214b6cb1">

### How to test

- Submit a webform with and without validation errors and check the dataLayer events

### Checklist




#### For all PR's


#### For new components only
